index.bs

```diff
@@ -2655,8 +2663,12 @@ The <dfn export for=commands>browser.createUserContext</dfn> command creates a
       <pre class="cddl remote-cddl">
       browser.CreateUserContext = (
         method: "browser.createUserContext",
-        params: EmptyParams,
+        params: browser.CreateUserContextParameters,
       )
+
+      browser.CreateUserContextParameters = {
+        ? acceptInsecureCerts: bool
```

Comment:
For now just a question to get a discussion started... For other features we decided against a parameter object for the `createUserContext` command and ended-up in having a `userContext` argument instead. This applies to `script.addPreloadScript` and `browsingContext.setViewport` so far. 

Adding parameters now would open up a different path in configuring user contexts. I find this a bit confusing and wonder if we should have some command for the network module to configure network behavior (similar like throttling, offline mode etc in the future).

Reply:
While `acceptInsecureCerts` is implementable via a dedicated command, the `proxy` setting is shared in the scope of the user context, and should be set at the moment the user context is created. So I believe we will need to extend it anyways. However, I can move the `acceptInsecureCerts` into `network` namespace in a dedicated command.

Reply:
the PR is updated

Reply:
Hi @sadym-chromium. We discussed it yesterday internally and agreed on that the former solution was better. Also it would match the same logic as setting the proxy settings which are per user context only. Would you mind reverting the last commit? Thanks!

Reply:
> Hi @sadym-chromium. We discussed it yesterday internally and agreed on that the former solution was better. Also it would match the same logic as setting the proxy settings which are per user context only. Would you mind reverting the last commit? Thanks!

Thanks git, no sweat ) done.